README.md

```diff
@@ -207,10 +207,12 @@ The acceptance tests will run against the Astronomer API and create/read/update/
 The Astro Terraform Import Script is a tool designed to help you import existing Astro resources into your Terraform configuration. 
 Currently, this script automates the process of generating Terraform import blocks and resource configurations for the following resources: workspaces, deployments, clusters, hybrid cluster workspace authorizations, API tokens, teams, team roles, and user roles.
 
-To use the import script, download the `import_script` executable file from [releases](https://github.com/astronomer/terraform-provider-astro/releases) and run it with the following command:
+To use the import script, download the `terraform-provider-astro-import-script` executable file from [releases](https://github.com/astronomer/terraform-provider-astro/releases) based on your OS and architecture and run it with the following command:
 
 ```
-./import [options]
+chmod +x terraform-provider-astro-import-script_<version-number>_<os>_<arc>
```

Comment:
this only applies to osx and linux i believe

Reply:
fixed - sep it for unix based and windows